### PR TITLE
Add Raheel to student

### PR DESCRIPTION
**Pull request by Raheel102**

### CODE REVIEW

**Comment by jigyasa-grover** [comment]
@Raheel102 Resolve merge conflicts.

**Comment by roonyh** [comment]
@Raheel102 more importantly you don't have to change anything in the index.html. Just edit .yml and github pages will do the rest :)

**Comment by jigyasa-grover** [comment]
@Raheel102 How is it going ?

**Comment by Raheel102** [comment]
Sorry I'm at school ill be on it as soon as I can

**Comment by jigyasa-grover** [comment]
Cool !

**Comment by Raheel102** [comment]
Yeah so all i need to do is change the .yml and get it to work on my github
pages and it will work?.... I wayyy overcomplicated it for myself whoops :D

**Comment by jigyasa-grover** [comment]
Yes, Just add a square picture of yours in the `img/students` folder and your details in `_data/students.yml` . That's it !

Also don't forget to make it as a single commit !

**Comment by Raheel102** [comment]
Must it be in one commit? i am having some trouble getting my pic to commit again

**Comment by jigyasa-grover** [comment]
Tried the instructions given [here](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) ?

**Comment by roonyh** [comment]
@Raheel102 Kudos for keep working on this!

Theres a bit of a problem with this one. See the files you have changed: https://github.com/fossasia/gci15.fossasia.org/pull/313/files

Whats marked red is stuff that you have removed. So you probably can see that you are removing some of the entries by other students. I am pretty sure you did not do this intentionally :) 
This could happen if your rebase operation did not go well.

So the easiest thing you can do at this moment, untill you learn more advanced git, is resetting everything and doing this all over again from the beginning. That would be the least complicated solution.

You can delete your fork and fork again and clone again if thats easy. Or you can go through https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md and follow instructions in it to do the reset.

**Comment by Raheel102** [comment]
I believe i am finally finished :D This was a very educational experience for which i am glad to have learnt more about using git

File: _data/students.yml [hunk 1]
```diff
@@ -369,3 +369,11 @@
   facebook: tanuj.rao
   twitter: tansanrao
   bio: A Sixteen Year Old Geek From India.
+
+
+ - name: Raheel Saghir
+  github: https://github.com/Raheel102
+  image: raheel102.jpg
+  facebook: https://www.facebook.com/raheel.mehmood.7
+  twitter: https://twitter.com/raheel102
```
Comment (roonyh) [inline _data/students.yml]: Please see other entries. We don't need urls.

**Comment by roonyh** [comment]
Good job! Remove the urls. Take a look at other students' entries. And your commit message should not be "Final". Add a commit message like "Add Raheel to the website"

**Comment by Raheel102** [comment]
HOW DO I OPEN IT BACK?

**Comment by Raheel102** [comment]
is there anyway i can reopen this? i accidentally closed it

**Comment by Raheel102** [comment]
sometimes repeating works...

File: _data/students.yml [hunk 2]
```diff
@@ -369,3 +369,10 @@
   facebook: tanuj.rao
   twitter: tansanrao
   bio: A Sixteen Year Old Geek From India.
+ 
+  name: Raheel Saghir
+   github: Raheel102
```
Comment (roonyh) [inline _data/students.yml]: Please fix the indentation.

**Comment by roonyh** [comment]
Good job! Did you mark your task to review?

**Pull request merged**